common-modules/src/main/java/com/softserve/if072/common/model/dto/AnalyticsProductDTO.java

```diff
@@ -0,0 +1,54 @@
+package com.softserve.if072.common.model.dto;
+
+/**
+ * The AnalyticsProductDTO class
+ *
+ * @author Igor Kryviuk
+ */
+public class AnalyticsProductDTO {
+    private int id;
+    private String name;
+    private String unit;
+
+    public AnalyticsProductDTO() {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1186)